lib/overcommit/hook/post_commit/git_guilt.rb

```diff
@@ -0,0 +1,9 @@
+module Overcommit::Hook::PostCommit
+  # Calculates the change in blame since the last revision.
+  class GitGuilt < Base
+    def run
+      system(executable, 'HEAD~', 'HEAD')
```

Comment:
Please use the `execute` helper for this, as it allows overcommit to properly handle setting up an appropriate environment for the child process to be spawned within.

```ruby
execute([executable, 'HEAD~', 'HEAD'])
```